<ui version="4.0" >
 <class>MetaContactSelectorWidget_Base</class>
 <widget class="QWidget" name="MetaContactSelectorWidget_Base" >
  <property name="geometry" >
        <rect>
            <x>0</x>
            <y>0</y>
            <width>427</width>
    <height>295</height>
        </rect>
    </property>
  <property name="sizePolicy" >
        <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>1</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
        </sizepolicy>
    </property>
  <property name="windowTitle" >
        <string>Select Contact</string>
    </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="lblHeader" >
     <property name="sizePolicy" >
                <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>4</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
        </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="lblSearch" >
       <property name="sizePolicy" >
                        <sizepolicy>
                            <hsizetype>1</hsizetype>
                            <vsizetype>5</vsizetype>
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
       <property name="text" >
                        <string>S&amp;earch:</string>
                    </property>
       <property name="buddy" >
                        <cstring>kListViewSearchLine</cstring>
                    </property>
                </widget>
     </item>
     <item>
      <widget class="Kopete::UI::ListView::SearchLine" name="kListViewSearchLine" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="Kopete::UI::ListView::ListView" name="metaContactListView" />
   </item>
  </layout>
        </widget>
  <customwidgets>
	<customwidget>
		<class>Kopete::UI::ListView::ListView</class>
   <extends></extends>
		<header>kopetelistview.h</header>
   <container>0</container>
   <pixmap></pixmap>
	</customwidget>
  <customwidget>
   <class>Kopete::UI::ListView::SearchLine</class>
   <extends></extends>
   <header>kopetelistviewsearchline.h</header>
   <container>0</container>
   <pixmap></pixmap>
	</customwidget>
 </customwidgets>
 <includes>
  <include location="global" >kopetelistviewsearchline.h</include>
  <include location="global" >kopetelistview.h</include>
  <include location="global" >q3header.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
